test/workflows/components/workflows.libsonnet

```diff
@@ -27,17 +27,12 @@
 
   // default parameters.
   defaultParams:: {
-    project:: "kubeflow-ci",
-    zone:: "us-east1-d",
     // Default registry to use.
-    registry:: "gcr.io/" + $.defaultParams.project,
+    registry:: "gcr.io/kubeflow-ci",
```

Comment:
Nit: after some underlying work done with Public ECR image, we can switch over to public ECR side, this should simplify WG folks' engineering work.

Reply:
Thanks! Should we wait until the public ECR image is ready?

Reply:
We can move forward on this PR currently, will need some changes after public ECR support is ready. 

I'll create another issue to keep track

Reply:
@PatrickXYS thanks!